tests/basic.feature

```diff
@@ -54,13 +54,13 @@ Feature: Testing Sphinx Writer
     Scenario: Indentation is ignored when any step in the scenario has text or a table
 
         Given a step with some text
-        '''
-        Here be that said text!
-        '''
+          '''
+          Here be that said text!
+          '''
```

Comment:
I'm still surprised to see indentation here...

Reply:
Meant to add a comment the last time you raised this question, but it got set aside for the larger rewrite. Long and short here is that `behave` will technically "do the right thing" with non-indented text, but `pytest-bdd` will only play nicely if the text is indented. From a visual scanning standpoint, indenting text attached to a step makes the most sense to logically/visually organize the data. Also, even `behave`'s [documentation on step text](https://behave.readthedocs.io/en/latest/gherkin.html#text) shows an indent in the example.

Reply:
Huh. Live'n'Learn. Thanks for the clarification!